pkgs/by-name/vl/vlc/package.nix

```diff
@@ -284,11 +287,21 @@ stdenv.mkDerivation (finalAttrs: {
   # should be the same as pkgsBuildBuild.qt5.qttranslations.
   postFixup = ''
     find $out/lib/vlc/plugins -exec touch -d @1 '{}' ';'
-    ${if stdenv.buildPlatform.canExecute stdenv.hostPlatform then "$out" else pkgsBuildBuild.libvlc}/lib/vlc/vlc-cache-gen $out/vlc/plugins
+    ${if stdenv.buildPlatform.canExecute stdenv.hostPlatform
+      then "$out"
+      else pkgsBuildBuild.libvlc}/lib/vlc/vlc-cache-gen $out/vlc/plugins
   '' + optionalString withQt5 ''
     remove-references-to -t "${libsForQt5.qtbase.dev}" $out/lib/vlc/plugins/gui/libqt_plugin.so
   '';
 
+  passthru = {
```

Comment:
We cannot use https://github.com/NixOS/nixpkgs/blob/5e05a5e3c91689cee2fe56ab7463a40b42d78990/doc/build-helpers/testers.chapter.md#check-the-program-version-using-a-specified-command-and-expected-version-string here?

Reply:
Yes. I will refactor this.